src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/CohostGoToDefinitionEndpointTest.cs

```diff
@@ -157,7 +160,7 @@ public partial class SurveyPrompt : ComponentBase
             """;
 
         var result = await GetGoToDefinitionResultAsync(input, FileKinds.Component,
-            (FileName("SurveyPrompt.razor"), surveyPrompt.Text),
+                (FileName("SurveyPrompt.razor"), surveyPrompt.Text),
```

Comment:
nit: extra indentation intended?

Reply:
No, thank you for pointing this out. I did move this, but I swear I reverted the change!

Seems I didn't have a good whitespace day yesterday, across the board  :)